scripts/publish/docs.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+'use strict';
+
+/* eslint-disable no-console */
+const fs = require('fs-extra');
+const path = require('path');
+const util = require('util');
+const exec = require('child_process').exec;
+
+const documentationPath = './docs/documentation';
+const outputPath = './tmp/';
+
+function createTempDirectory() {
+  return new Promise((resolve, reject) => {
+    fs.emptyDir(outputPath, (error) => {
```

Comment:
Please use `tmp`, which creates temporary directory outside the project that will be deleted by the OS. See how we create a temporary directory for our e2e tests.